app/helpers/check_answers_helper.rb

```diff
@@ -1,16 +1,34 @@
 module CheckAnswersHelper
   include GovukLinkHelper
 
-  def display_answered_questions_summary(subsection, case_log)
-    total = subsection.applicable_questions_count(case_log)
-    answered = subsection.answered_questions_count(case_log)
+  def display_answered_questions_summary(subsection, case_log, current_user)
+    total = total_count(subsection, case_log, current_user)
+    answered = answered_questions_count(subsection, case_log, current_user)
     if total == answered
       '<p class="govuk-body">You answered all the questions.</p>'.html_safe
     else
       "<p class=\"govuk-body\">You have answered #{answered} of #{total} questions.</p>".html_safe
     end
   end
 
+private
+
+  def answered_questions_count(subsection, case_log, current_user)
```

Comment:
These move out of the class and into the helper because they're currently only used for view logic, no where else. (i.e. they're actually specifically for the check answers page).